geode-core/src/main/java/org/apache/geode/internal/cache/tx/RemotePutMessage.java

```diff
@@ -311,10 +312,11 @@ public static boolean distribute(EntryEventImpl event, long lastModified, boolea
         }
         successful = true; // not a cancel-exception, so don't complain any more about it
 
-      } catch (RemoteOperationException e) {
+      } catch (RegionDestroyedException | RemoteOperationException e) {
         if (logger.isTraceEnabled(LogMarker.DM_VERBOSE)) {
           logger.trace(LogMarker.DM_VERBOSE,
-              "RemotePutMessage caught an unexpected exception during distribution", e);
+              "RemotePutMessage caught an exception during distribution; retrying to another member",
```

Comment:
Why are we retrying if the region is destroyed? it will see the same exception, after sending it to the another recipient, isn't it.

Reply:
If cache is closed on one of the node, we may get CacheClosedException or RegionDestroyedException. This does not mean region is destroyed on other member.